modules/dnn/src/layers/padding_layer.cpp

```diff
@@ -34,7 +34,12 @@ class PaddingLayerImpl CV_FINAL : public PaddingLayer
     PaddingLayerImpl(const LayerParams &params)
     {
         setParamsFrom(params);
-        paddingValue = params.get<float>("value", 0);
+        if (params.blobs.size() == 1)
+            // ONNX parser supports any type
+            paddingValue = params.blobs[0];
+        else
+            // Other parsers support only float
+            paddingValue = Mat(1, 1, CV_32F, params.get<float>("value", 0));
```

Comment:
Looks weird to me. Now it is a `Mat` either way. Possible to change the behavior of other importers as well?

Reply:
tf_lite importer adds some blobs to any layer (probably constant inputs)
https://github.com/opencv/opencv/blob/4.x/modules/dnn/src/tflite/tflite_importer.cpp#L293
And there is a test where padding layer already has one blob. So looks like storing padding value in a blob requires some significant changes in the tf_lite parser.

Reply:
@dkurt Could you verify this?

Reply:
TFLite padding layer does not set padding values at all: https://github.com/opencv/opencv/blob/52f3f5a3f617e8b1fa708bdfc37a6bc58a13ebb7/modules/dnn/src/tflite/tflite_importer.cpp#L479